backend/migrations/versions/b833ad41db68_maintenance_window_schema.py

```diff
@@ -0,0 +1,74 @@
+"""maintenance_window_schema
+
+Revision ID: b833ad41db68
+Revises: 194608b1ff7f
+Create Date: 2024-04-16 19:30:05.226603
+
+"""
+
+import os
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy import Column, String, orm
+
+from dataall.base.db import get_engine, has_table, Base
+
```

Comment:
Can you merge `main` into this branch? I think we will face some migration conflicts

Reply:
Will do sure!!